test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -439,20 +436,28 @@ class test_uct_cm_sockaddr : public uct_test {
             self->server_accept(self->m_server, conn_request,
                                 server_connect_cb, server_disconnect_cb, self);
         }
+        UCS_TEST_MESSAGE << "Connection requested";
+        self->m_cm_state |= TEST_CM_STATE_CONNECT_REQUESTED;
+        self->m_server_recv_req_cnt++;
     }
 
     static void
     server_connect_cb(uct_ep_h ep, void *arg, ucs_status_t status) {
-         test_uct_cm_sockaddr *self = reinterpret_cast<test_uct_cm_sockaddr *>(arg);
-         self->m_cm_state |= TEST_CM_STATE_SERVER_CONNECTED;
-         self->m_server_connect_cb_cnt++;
+        test_uct_cm_sockaddr *self = reinterpret_cast<test_uct_cm_sockaddr *>(arg);
+
+        UCS_TEST_MESSAGE << "Server connected " << ucs_status_string(status);
+        EXPECT_EQ(UCS_OK, status);
+        self->m_cm_state |= TEST_CM_STATE_SERVER_CONNECTED;
+        self->m_server_connect_cb_cnt++;
     }
 
     static void
     server_disconnect_cb(uct_ep_h ep, void *arg) {
         test_uct_cm_sockaddr *self = reinterpret_cast<test_uct_cm_sockaddr *>(arg);
+        UCS_TEST_MESSAGE << "Server disconnected";
 
         if (!(self->m_server_start_disconnect)) {
+            UCS_TEST_MESSAGE << "Server disconnecting from disconnect callback";
```

Comment:
too many prints from the test.. maybe ucs_debug() instead of UCS_TEST_MESSAGE

Reply:
removed prints from all callbacks